src/components/tickets/ticket.service.ts

```diff
@@ -1,8 +1,26 @@
 import { Ticket } from './ticket'
 import { Request, Response, NextFunction } from 'express'
+import markdown from 'markdown-it'
+import sanitizeHtml from 'sanitize-html'
+
+function formatMdToSafeHTML(mdBody: string): string {
+  const dirtyHtml = new markdown().render(mdBody)
+  const cleanHtml = sanitizeHtml(dirtyHtml, {
+    transformTags: {
+      'h1': 'h3',
+      'h2': 'h4',
+      'h3': 'h5',
+      'h4': 'h5'
+    }
+  })
+  return cleanHtml
+}
 
 export const getTickets = async (req: Request, _res: Response, next: NextFunction) => {
   req.tickets = await Ticket.query().orderBy('createdAt', 'ASC')
+  req.tickets.forEach((ticket) => {
+    ticket.description = formatMdToSafeHTML(ticket.description)
+  })
```

Comment:
This I think can be written like this:
```
req.tickets = (await Ticket.query().orderBy('createdAt', 'ASC')).map(ticket => {
  ticket.description = formatMdToSafeHTML(ticket.description)
  return ticket
}
```
I think using a `map` may be a little more descriptive in this case.

Reply:
Completely forgot about .map. Thanks!